l10n/en/products/monitor/breaches.ftl

```diff
@@ -0,0 +1,71 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/monitor/breaches/
+
+-brand-HIBP = Have I Been Pwned
+
+# HTML all breaches page title
+monitor-breaches-page-title = { -brand-name-mozilla-monitor } — All Data Breaches
+
+# HTML detail breach page title
+# Variables:
+#   $breach_title (string) - the title of the breach
+monitor-breach-page-title = { -brand-name-mozilla-monitor } — { $breach_title }
+
+# HTML page description
+monitor-breaches-page-desc = Browse the complete list of known breaches detected by { -brand-name-mozilla-monitor }, then find out if your information was exposed.
+
+monitor-breaches-hero-heading = All breaches detected by { -brand-name-mozilla-monitor }
+monitor-breaches-hero-tagline = We monitor all known data breaches to find out if your personal information was compromised. Here’s a complete list of all of the breaches that have been reported since 2007.
+
+breach-added-label = Breach added
+breach-exposed-data = Exposed data
```

Comment:
Nit: for FTL strings we should try and start all strings in a file with a common naming prefix, such as `monitor-breaches-`
```suggestion
monitor-breaches-added-label = Breach added
monitor-breaches-exposed-data = Exposed data
```